pbspro.spec

```diff
@@ -39,25 +39,35 @@
 %if !%{defined pbs_name}
 %define pbs_name pbspro
 %endif
+
 %if !%{defined pbs_version}
-%define pbs_version %(/bin/ls -1t %{_sourcedir}/%{pbs_name}-*.tar.gz | head -1 | sed -n 's:.*/pbspro-\\(.*\\).tar.gz:\\1:p')
+%define pbs_version %(
```

Comment:
Thanks for the suggestion. I'm trying to account for the possibility of multiple pbspro-*.tar.gz files being present in _sourcedir. If the wildcard matches multiple files, it will generate an error.